config/owasp/suppressions.xml

```diff
@@ -211,6 +211,7 @@
    ]]></notes>
         <packageUrl regex="true">^pkg:npm/ip@.*$</packageUrl>
         <vulnerabilityName>1096432</vulnerabilityName>
+        <vulnerabilityName>1096460</vulnerabilityName>
```

Comment:
may be try updating with v1.1.8? Seems to be the latest/stable version (weird though as it is after 2.0.0)

https://www.npmjs.com/package/ip?activeTab=versions